internal/crypto/tls.go

```diff
@@ -28,7 +28,7 @@ func TLSConfigForServer(caConfig, serverConfig *TLSCertificateConfig) (*tls.Conf
 
 	tlsConfig := &tls.Config{
 		Certificates: []tls.Certificate{cert},
-		MinVersion:   tls.VersionTLS13,
+		MinVersion:   tls.VersionTLS10,
```

Comment:
AAP Gateway did not support TLS13, we will need to make this configurable

Reply:
I hope AAP GW can be updated, but if not, we really should fall below `VersionTLS12`...